plugins/ooni/main.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "OONI",
+    "short": "Open Observatory of Network Interference",
+    "description": "A free software, global observation network for detecting censorship, surveillance and traffic manipulation on the internet",
+    "version": "1.0.1",
+    "exec": [
+    {
+        "full-path": "/usr/bin/traceroute"
```

Comment:
The program doesn't currently make any use of the binary's paths, so I haven't changed them yet. Just wanted to tested if multiple installed plugins were displayed properly.